spec/ffmpeg/movie_spec.rb

```diff
@@ -277,7 +277,7 @@ module FFMPEG
         end
 
         it "should parse the bitrate" do
-          expect(@movie.bitrate).to eq(481836)
+          expect(@movie.bitrate).to be_between(481736, 481936)
```

Comment:
This test appears to be flakey, bitrate isn't always exactly the same so we just require an approximation.